modules/virt-features-for-storage-matrix.adoc

```diff
@@ -13,11 +13,12 @@ endif::[]
 
 .{VirtProductName} storage feature matrix
 ifdef::virt-features-for-storage[]
-[cols="40%,30%,30%",options="header"]
+[cols="40%,20%,20%,20%",options="header"]
```

Comment:
So... I don't really know what to say here. I know that this table was already kind of like this, but this seems like a nightmare to maintain.

I don't think we're saving any time by trying to combine this into one table with a bunch of ifdef/ifeval statements. Especially with there being a different number of columns. Couldn't this just be two (or more) separate tables?

It's really hard for me to review these changes as is. But I think that when I look at this table here [1], that there aren't any callouts listed below like there should be: https://bz1890663--ocpdocs.netlify.app/openshift-enterprise/latest/virt/virtual_machines/importing_vms/virt-importing-rhv-vm.html#openshift-virtualization-prerequisites

Reply:
Hi @bergerhoffer -

I know the conditionals are dense... I did not create them. it wouldn't be my preference either, to be honest, but it's what we have to deal with for this release.

This was designed as a table to have footnotes, not callouts. I worked with @adellape to do the original design. if we need to modify it, I'd prefer to defer that to a future release. @adellape --- Should we change these to callouts?

Thanks
Bob

Reply:
Hey, yeah I totally understand that it's outside the scope of this PR to update the entire table, just sharing that you might want to consider in the future because it's hard to review and update. And I'm fine with the format of the footnotes, it's the fact that they aren't showing at all whenever I look at the section that I linked.

Reply:
> it's the fact that they aren't showing at all whenever I look at the section that I linked.

Ah, OK, this is Avital's table you are linking to. It could be I screwed that up. Tagging @apinnick to see if I did mess up her presentation and if so, what I need to do to fix it.

Reply:
This has been resolved. All three flavors of this table display as intended: 

https://bz1890663--ocpdocs.netlify.app/openshift-enterprise/latest/virt/virtual_machines/virtual_disks/virt-features-for-storage.html
https://bz1890663--ocpdocs.netlify.app/openshift-enterprise/latest/virt/virtual_machines/importing_vms/virt-importing-rhv-vm.html
https://bz1890663--ocpdocs.netlify.app/openshift-enterprise/latest/virt/virtual_machines/importing_vms/virt-importing-vmware-vm.html